site/index.qmd

```diff
@@ -7,86 +7,89 @@ resources:
-    sort: false
-    fields: [title, description]
-    contents:
-    - /guide/model-validation/preparing-validation-reports.qmd
-    - /guide/model-validation/working-with-model-findings.qmd
-    - /guide/reporting/working-with-analytics.qmd
-  - id: releases
-    type: grid
-    grid-columns: 3
-    max-description-length: 150
-    sort: false
-    fields: [title, subtitle, description]
-    contents:
+# listing:
+  # - id: admin
+  #   type: grid
+  #   grid-columns: 1
+  #   max-description-length: 250
+  #   sort: false
+  #   fields: [title, description]
+  #   contents:
+  #   - /guide/configuration/accessing-validmind.qmd
+  #   - /guide/configuration/managing-your-organization.qmd
+  #   - /guide/configuration/managing-users.qmd
+  # - id: developer
+  #   type: grid
+  #   grid-columns: 1
+  #   max-description-length: 250
+  #   sort: false
+  #   fields: [title, description]
+  #   contents:
+  #   - /developer/validmind-library.qmd
+  #   - /developer/supported-models.qmd
+  #   - /developer/samples-jupyter-notebooks.qmd
+  # - id: validator
+  #   type: grid
+  #   grid-columns: 1
+  #   max-description-length: 250
+  #   sort: false
+  #   fields: [title, description]
+  #   contents:
+  #   - /guide/model-validation/preparing-validation-reports.qmd
+  #   - /guide/model-validation/working-with-model-findings.qmd
+  #   - /guide/reporting/working-with-analytics.qmd
+  # - id: releases
+  #   type: grid
+  #   grid-columns: 3
+  #   max-description-length: 150
+  #   sort: false
+  #   fields: [title, subtitle, description]
+  #   contents:
     # MAKE-RELEASE-NOTES-LATEST-MARKER
-      - /releases/2025/2025-apr-24/release-notes.qmd
-      - /releases/2025/2025-mar-07/release-notes.qmd
-      - /releases/2025/2025-jan-31/release-notes.qmd
+      # - /releases/2025/2025-apr-24/release-notes.qmd
+      # - /releases/2025/2025-mar-07/release-notes.qmd
+      # - /releases/2025/2025-jan-31/release-notes.qmd
     # MAKE-RELEASE-NOTES-OLDEST-MARKER
-  - id: validmind-academy
-    type: grid
-    image-height: 200px
-    sort: false
-    contents:
-    - path: training/administrator-fundamentals/administrator-fundamentals-register.qmd
-      image: training/administrator-fundamentals/administrator-fundamentals.png
-      title: "Administrator Fundamentals {{< fa gear >}}"
-      subtitle: "Register now {{< fa chevron-right >}}"
-    - path: training/developer-fundamentals/developer-fundamentals-register.qmd
-      title: "Developer Fundamentals {{< fa code >}}"
-      subtitle: "Register now {{< fa chevron-right >}}"
-      image: training/developer-fundamentals/developer-fundamentals.png
-    - path: training/validator-fundamentals/validator-fundamentals-register.qmd
-      image: training/validator-fundamentals/validator-fundamentals.png
-      title: "Validator Fundamentals {{< fa user-check >}}"
-      subtitle: "Register now {{< fa chevron-right >}}"
-    fields: [image, title, subtitle]
+  # - id: validmind-academy
```

Comment:
Similar comment here about keeping things tidy and free of cruft: if this content is no longer needed, delete it, please.